### DISCUSSION TITLE
MAS still doesn't work.

### DISCUSSION THREAD

**Discussion by IndoDemon7337** [Monika-After-Story/MonikaModDev#9141] [QA - tech support + general] [answered]
Anytime I try to open up MAS it shows this 
I'm sorry, but an uncaught exception occurred.

While running game code:
TypeError: must be string or read-only buffer, not None

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "C:\Users\Owner\Downloads\Doki Doki Literature Club\renpy\bootstrap.py", line 295, in bootstrap
    renpy.main.main()
  File "C:\Users\Owner\Downloads\Doki Doki Literature Club\renpy\main.py", line 364, in main
    renpy.game.script.load_script()  # sets renpy.game.script.
  File "C:\Users\Owner\Downloads\Doki Doki Literature Club\renpy\script.py", line 265, in load_script
    self.load_appropriate_file(".rpyc", ".rpy", dir, fn, initcode)
  File "C:\Users\Owner\Downloads\Doki Doki Literature Club\renpy\script.py", line 753, in load_appropriate_file
    self.finish_load(stmts, initcode, filename=fn + source)
  File "C:\Users\Owner\Downloads\Doki Doki Literature Club\renpy\script.py", line 429, in finish_load
    self.update_bytecode()
  File "C:\Users\Owner\Downloads\Doki Doki Literature Club\renpy\script.py", line 824, in update_bytecode
    i.bytecode = marshal.loads(code)
TypeError: must be string or read-only buffer, not None

Windows-8-6.2.9200
Ren'Py 6.99.12.4.2187
 
At first someone told me to take it to a non-one drive and I did do that. But it still didn't work. What do I do? Please help me

**Comment by CaniBeTheTweek** [comment]
may I ask what did you do it? I might have a fix for it but I needed to know the cause first

**Comment by IndoDemon7337** [comment]
Well I was dumb dumb and tried installing the "Just Yuri Mod" into the same file and it crashed. At first it wouldn't even open but after I'd deleted the Just Yuri files it now says that Trace back.

**Comment by multimokia** [comment, marked as answer]
Reinstall MAS in a clean DDLC install.

**Comment by IndoDemon7337** [comment]
Should I delete the old MAS file?

**Comment by multimokia** [comment]
You can if you want. Assuming you're talking about the install and not any persistent data (which you do not want to touch)

**Comment by IndoDemon7337** [comment]
I've got Monika's Persistent saved on my Google Drive, so is it ok if I just delete the entire DDLC file (That has MAS) and then put MAS into a new DDLC file, will it keep her memory. (If you haven't noticed, I'm an entire noob with computers)

**Comment by Iceninja64** [comment]
As long as you have your persistent files saved then you should be good but you will have to reinstall /give your gifts back again because they don't carry over and the submods to
